apps/web/src/components/OverviewRow.tsx

```diff
@@ -29,26 +31,37 @@ export const OverviewRow = ({
     0,
     Math.min(4, formInstances.length),
   );
+
   return (
     <>
       <Box w={rowWidth}>
         <Flex justifyContent="space-between" alignItems="center">
           <Flex alignItems="center">
-            <Heading as="h2">{title}</Heading>
-            <Flex
-              marginLeft="13px"
-              backgroundColor={color}
-              height="18px"
-              width="32px"
-              borderRadius="12"
-              justifyContent="center"
-              alignItems="center"
-            >
-              <Text fontSize="14px" fontWeight="700" color="#756160">
-                {formInstances.length}
-              </Text>
-            </Flex>
+            <Heading as="h2">
+              {title == 'To Do'
+                ? `You have ${formInstances.length} forms waiting for you.`
+                : title}
+            </Heading>
+
+            {title != 'To Do' ? (
```

Comment:
Can just do `title != 'To Do` && ...` so you don't need to do the colon and empty tags for the other case

Reply:
Or you can just move this flex to the conditional above. Up to you.